tests/main.sh

```diff
@@ -162,7 +162,7 @@ while getopts "hH?vVtAs:a:x:rl:p:S:" opt; do
         export BOOTSTRAP_REUSE_LOCAL="${OPTARG}"
         export BOOTSTRAP_REUSE="true"
         CLOUD=$(juju show-controller "${OPTARG}" --format=json | jq -r ".[\"${OPTARG}\"] | .details | .cloud")
-        PROVIDER=$(juju clouds --client 2>/dev/null | grep "${CLOUD}" | awk '{print $4}')
+        PROVIDER=$(juju clouds --client 2>/dev/null | grep "${CLOUD}" | awk '{print $4}' | head -n 1)
```

Comment:
Drive-by fix from @achilleasa recent changes, if you have multiple LXD clouds, this breaks.